string hostname

Cpu[] cpus
int32 running_processes
float32 cpu_usage_overall

int64 memory_available
int64 memory_used
int64 memory_total

Filesystem[] filesystems
NetworkInterface[] network_interfaces

